pkg/tsdb/prometheus/buffered/time_series_query.go

```diff
@@ -362,14 +362,14 @@ func matrixToDataFrames(matrix model.Matrix, query *PrometheusQuery, frames data
 			tags[string(k)] = string(v)
 		}
 		timeField := data.NewFieldFromFieldType(data.FieldTypeTime, len(v.Values))
-		valueField := data.NewFieldFromFieldType(data.FieldTypeNullableFloat64, len(v.Values))
+		valueField := data.NewFieldFromFieldType(data.FieldTypeFloat64, len(v.Values))
```

Comment:
@ryantxu there is this one difference that I am not sure where to align. Here in the new code https://github.com/grafana/grafana/blob/f4f31b40fd9003ae87896eb328795d40ec6b1a9c/pkg/util/converter/prom.go#L480 we make it non nullable, but is it right? Are values from Prometheus always non null?

Reply:
Results from prometheus will not be null (but they may be NaN)

Reply:
Ok so I will keep the change here.

Reply:
Looks like this change introduced a change when dividing zero by zero: https://github.com/grafana/grafana/issues/54857

Reply:
@aocenas Is reverting this back (to `FieldTypeNullableFloat64`) going to hurt anything?

Reply:
Well I think maybe this is for @ryantxu? It seems like this can be null in the end? In that case the part in the new client should be also changed to `FieldTypeNullableFloat64`. This was just to align the new client path and the old client path as they ended up creating a bit different dataFrames.

Reply:
Based no your comment here https://github.com/grafana/grafana/issues/54857#issuecomment-1244173032 it seems like yeah this should be `FieldTypeNullableFloat64` here and also in the new client code to align it.